### PR TITLE
Fix #1415 : Added CFEP in the Glossary. 

### LINKED ISSUE #1415: Add  CFEP in Glossary
`CFEP` : conda-forge enhancement proposals.

Since all the major changes in the projects are submitted as CFEP.  It's a prominent keyword as well. So what I think we can add this keyword in the Glossary section, to help the readers know about it. The keyword is used in docs many a time so to make that clear what CFEP exactly is we can add that in the Glossary.
Reference:  https://github.com/conda-forge/cfep

Not sure if it's gonna help, I would like to know your views on it.

### PR DESCRIPTION
**Pull request by prachi237**
Fixes: #1415 
- [x] make all edits to the docs in the `src` directory, not in `docs` or in the html files
- [x] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [x] put any other relevant information below

### CODE REVIEW

File: src/misc/00_intro.rst [hunk 1]
```diff
@@ -24,6 +24,8 @@ Glossary
   CRAN
     **C**\ omprehensive **R** **A**\ rchive **N**\ etwork. CRAN is a network of FTP and web servers around the world that store identical, up-to-date, versions of code and documentation for R. `Learn More <https://cran.r-project.org/>`_.
 
+  CFEP  
+    **C**\ onda **F**\ orge **E**\ nhancement **P**\ roposal. This CFEP contains the index of all conda-forge Enhancement Proposals, known as CFEPs. All major changes to the project should be submitted as CFEP.
```
Comment (viniciusdc) [inline src/misc/00_intro.rst]:
```suggestion
    **C**\ onda **F**\ orge **E**\ nhancement **P**\ roposal. As conda-forge grows, all major changes to the project from a technical standpoint as well as to address social topics such as governance and expected conduct should be submitted as a CFEP. `Learn More <https://conda-forge.org/docs/orga/cfep-index.html/>`_.
```

**Review by viniciusdc — COMMENTED** [review]
I just put more info about what changes should a CFEP contain as also a direct link to our org CFEP' list

File: src/misc/00_intro.rst (hunk 1 above)
Comment (prachi237) [reply src/misc/00_intro.rst]: I  tried to make that short,😅 my bad its better to add the same as mentioned in  https://github.com/conda-forge/cfep
Thanks @viniciusdc

Comment (viniciusdc) [reply src/misc/00_intro.rst]: due to its contents and how it changes conda-forge, the acceptance of a CFEP depends on a voting session from core.

Comment (viniciusdc) [reply src/misc/00_intro.rst]: > I tried to make that short, my bad its better to add the same as mentioned in https://github.com/conda-forge/cfep
> Thanks @viniciusdc

no worries it is good to be brief too, I just find this line "This CFEP contains the index of all conda-forge Enhancement Proposals" quite strange from the rest of the text, as it seans you are refering to an specifq CFEP (or an index containing then, and that's not what a CFEP means)

File: src/misc/00_intro.rst [hunk 2]
```diff
@@ -24,6 +24,8 @@ Glossary
   CRAN
     **C**\ omprehensive **R** **A**\ rchive **N**\ etwork. CRAN is a network of FTP and web servers around the world that store identical, up-to-date, versions of code and documentation for R. `Learn More <https://cran.r-project.org/>`_.
 
+  CFEP  
+    **C**\ onda **F**\ orge **E**\ nhancement **P**\ roposal. As conda-forge grows, all major changes to the project from a technical standpoint as well as to address social topics such as governance and expected conduct should be submitted as a CFEP. `Learn More <https://conda-forge.org/docs/orga/cfep-index.html#index-of-cfeps/>`_.
```
Comment (viniciusdc) [inline src/misc/00_intro.rst:28]:
```suggestion
    **C**\ onda **F**\ orge **E**\ nhancement **P**\ roposal. All major changes to the conda-forge project, from a technical standpoint as well as to address social topics such as governance and expected conduct, should be submitted as a CFEP. `Learn More <https://conda-forge.org/docs/orga/cfep-index.html#index-of-cfeps/>`_. 
```

File: src/misc/00_intro.rst (hunk 1 above)
Comment (prachi237) [reply src/misc/00_intro.rst]: ahhh actually I took the reference from https://www.python.org/dev/peps/  as this too works in the same way. But that's true it's not conveying the purpose

**Review by viniciusdc — APPROVED** [review]
LGTM

**Review by beckermr — CHANGES_REQUESTED** [review]
Please consult our governance docs and cfeps 0 and 1 for the scope and purpose of cfeps. Governance changes in practice do not go through the cfep process.

**Comment by viniciusdc** [comment]
> Please consult our governance docs and cfeps 0 and 1 for the scope and purpose of cfeps. Governance changes in practice do not go through the cfep process.

It was my bad, I just followed https://github.com/conda-forge/cfep. We might need to update that as well then

**Comment by beckermr** [comment]
@viniciusdc this needs to be fixed

**Comment by prachi237** [comment]
> @viniciusdc this needs to be fixed
 @viniciusdc  sir, from where shd I take the reference.

**Comment by viniciusdc** [comment]
> @viniciusdc this needs to be fixed

Now I'm kind of confused, CFEP-01 says: These changes may be technical in manner or may deal with social aspects of the project including governance and conduct. In the very first section: https://github.com/conda-forge/cfep/blob/master/cfep-01.md#what-is-a-cfep

**Comment by beckermr** [comment]
The definition says "All major changes...". We don't require that.

**Comment by prachi237** [comment]
Shall I add this definition:
A CFEP is a document that outlines a suggested change to how the project operates. These changes may be technical in manner or may deal with social aspects of the project including governance and conduct.

**Comment by viniciusdc** [comment]
https://github.com/conda-forge/conda-forge.github.io/pull/1417

**Comment by beckermr** [comment]
Yup that looks great!

**Comment by viniciusdc** [comment]
> Yup that looks great!

Thanks for the review Matt, sorry for this mistake

**Pull request merged**